app/server/utils/credit_transfer.py

```diff
@@ -77,12 +81,12 @@ def calculate_transfer_stats(total_time_series=False, start_date=None, end_date=
 
     beneficiary_filters = [User.has_beneficiary_role == True]
 
-    total_beneficiaries = db.session.query(User).filter(*beneficiary_filters).filter(*dateFilter).count()
+    total_beneficiaries = db.session.query(User).filter(*beneficiary_filters).filter(*userDateFilter).count()
```

Comment:
was using a credittransfer filter on user table which was causing the error @tristanhcole found and noted on slack. 

when a daterange filter is applied it, the usage funnel will filter for newly created users/beneficiaries in the specified period. this may or may not be what we actually want for that section but it will prevent the randomly large number error from appearing